web/packages/teleterm/src/services/appUpdater/appUpdater.ts

```diff
@@ -107,11 +304,48 @@ export class AppUpdater {
       managingClusterUri,
       getClusterVersions: this.getClusterVersions,
     });
-    if (this.autoUpdatesStatus.enabled) {
-      autoUpdater.autoDownload = shouldAutoDownload(this.autoUpdatesStatus);
-    }
     this.logger.info('Resolved auto updates status', this.autoUpdatesStatus);
   }
+
+  /**
+   * Workaround to prevent installing outdated updates.
+   * electron-updater lacks support for this: once an update is downloaded,
+   * it will be installed on quit—even if subsequent update checks report
+   * no new updates.
+   */
+  private async preventInstallingOutdatedUpdates(): Promise<void> {
+    if (!this.isUpdateDownloaded) {
```

Comment:
Thoughts on inverting this and returning early to bring the rest of the function less indented?

Reply:
Absolutely, inverted.